bot.py

```diff
@@ -72,6 +72,15 @@ async def on_command_error(ctx, error):
         await log(client, f'{author} attempted to run `{message}` but failed because of an unexpected error: {error}')
 
 
+@client.event
+async def on_message(message):
+    if message.author == client.user:
+        return
+    channel = message.channel
+    if channel.name == 'i-made-a-poppy':
```

Comment:
There was a typo in the original issue, the channel name here should be 'i-made-a-poopy'

Reply:
I'd say we make a new cog for listeners